api/resolvers/wallet.js

```diff
@@ -111,17 +111,79 @@ export default {
       }
 
       if (include.has('stacked')) {
+        // query1 - get all sats stacked for OP with no forwards in play at all
```

Comment:
Some of the accumulation logic varies by different `AND` conditions in the `WHERE` clauses, so I found it easier to write them in separate queries than to merge them all together. This shouldn't require extra round trips to the DB though, right? Since they're all `UNION`ed together later and executed as one large query?

Reply:
No extra round trips, you're right. It's just more code to reason through.

Reply:
It's largely just a relic of how I iterated on the queries during development - I targeted one use-case at a time and built each query independently. I found it easier to manage that way. I can consolidate them into one if you'd like, LMK.

Reply:
Up to you. I'll probably consolidate them if you don't.

Reply:
I left them split up, I personally found them easier to manage this way. Feel free to consolidate though.